app/javascript/components/forms/profile/component.jsx

```diff
@@ -77,55 +86,108 @@ class ProfileForm extends PureComponent {
                         validate={[validateEmail]}
                         required
                       />
-                      <Select
-                        name="language"
-                        label="language"
-                        options={getLanguages()}
-                        placeholder="Select a language"
-                        required
-                      />
                       <Select
                         name="sector"
                         label="sector"
-                        options={sectors.map(s => ({ label: s, value: s }))}
-                        placeholder="Select a language"
-                        required
+                        options={Object.keys(sectors).map(s => ({
+                          label: s,
+                          value: s
+                        }))}
+                        placeholder="Select a sector"
+                        required={
+                          source === 'AreaOfInterestModal' ? true : undefined
```

Comment:
never set a variable to `undefined` by yourself - this is reserved to javascript engine - because when debugging you cannot tell if the value is undefined because you never defined value and js engine set it to undefined (all vars in js are initially set to undefined) or if you did somewhere. it can make debugging very confusing :P

Reply:
especially here, this can be solved with `required={source === 'AreaOfInterestModal'}`, if the source is something else, it's gonna evaluate to false

Reply:
So in this case required could be false as you said. I'm using undefined because it's the same thing as not passing the prop 'required'. Ugly but that's how it is. I'm doing this to let the library manage if it's required or not, but for now false would be more than enough, you're right 👍